crates/e2e/tests/e2e/flashloans.rs

```diff
@@ -200,21 +213,16 @@ async fn forked_mainnet_repay_debt_with_collateral_of_safe(web3: Web3) {
         }
     };
 
-    // pay some extra for the flashloan fee
-    let fee_bps = aave_pool.flashloan_premium_total().call().await.unwrap();
-    let flashloan_fee = (flashloan_amount * U256::from(fee_bps)).ceil_div(&10_000.into());
     let mut order = OrderCreation {
         sell_token: usdc.address(),
         sell_amount: collateral_amount,
-        buy_token: onchain.contracts().weth.address(),
+        buy_token: weth.address(),
+        // we want to get exactly enough WETH to repay the flashloan
         buy_amount: flashloan_amount + flashloan_fee,
         valid_to: model::time::now_in_epoch_seconds() + 300,
         kind: OrderKind::Buy,
         app_data,
         partially_fillable: false,
-        // receiver is always the settlement contract because the driver takes
-        // funds from the settlement contract to pay back the loan
-        receiver: Some(onchain.contracts().gp_settlement.address()),
```

Comment:
The restriction that all flashloan orders have to be buy orders with `receiver == settlementContract` will also be lifted in the latest iteration of the flashloan implementation.